README.md

```diff
@@ -213,25 +213,24 @@ to appeal.
 
 ## Fees
 
-The dex does not charge trading fees, but users have to pay on-chain transaction
-fees. Unlike on centralized exchanges where users can trade any arbitrary amount
-of funds, orders on the dex must be made in increments of a specified lot size.
-For example, if the lot size for the DCR/BTC market is 40 DCR, users can
-make orders to buy or sell DCR in increments of 40.
-
-You might be lucky and have your entire order matched by one other user. In
-this case, you will have to pay 2 transaction fees: one for sending funds on
-one chain, and one for receiving funds on the other. In the worst case, each
-lot of your order is matched by a different user, and you will pay a separate
-sending and recieving transaction for each match. Check the 
+The dex does not charge fees, but users pay on-chain transaction fees.
+
+To ensure that on-chain transaction fees do not eat a significant portion of the
+order quantity, orders must be specified in increments of a minimum lot size.
+For example, if on-chain transaction fees worked out to $5, and a user was able
+to place an order to trade $10, they would lose half of their trade to
+transaction fees. So for high-fee assets, the operator must set a large lot
+size.
+
+The best case scenario is for an entire order be consumed by a single match.
+If this happens, the user pays 2 transaction fees: one swap and one redemption.
+The worst case is for the order to be consumed by single lot matches per epoch,
```

Comment:
single lot matches per epoch -> one single-lot match per epoch

Just to explain, you can have more than one single-lot match in an epoch if you match with different orders. In that case, matches can be grouped, so it's no longer worst-case.

Reply:
Got it.

Reply:
Correct it's no longer the worst-case, but not best either given the txn size is bigger.  Referring to discussion in https://github.com/decred/dcrdex/pull/1074#discussion_r645634303

The per-epoch thing is also somewhat irrelevant for taker orders, as I've illustrated there.